README.md

```diff
@@ -1,81 +1,63 @@
-# LDAP-sync
+# Famedly Sync
 
-Sync LDAP/AD users with Zitadel.
+This tool synchronizes users from different sources to Famedly's Zitadel instance.
 
-## Usage
+Currently supported sources:
+- LDAP
+- Custom endpoint provided by UKT
 
-The easiest way to deploy this tool is using our published docker
-container through our [composefile](./docker-compose.yaml).
+## Configuration
 
-To prepare for use, we need to provide a handful of files in an `opt`
-directory in the directory where `docker compose` will be
-executed. This is the expected directory structure of the sample
-configuration file:
+The tool expects a configuration file located at `./config.yaml`. See example configuration at [config.sample.yaml](./config.sample.yaml).
 
-```
-opt
-├── certs
-│  └── test-ldap.crt   # The TLS certificate of the LDAP server
-├── config.yaml        # An example can be found in config.sample.yaml
-└── service-user.json  # Provided by famedly
-```
+The default path can be changed by setting the new path to the environment variable `FAMEDLY_LDAP_SYNC_CONFIG`.
 
-Once this is in place, the container can be executed in the parent
-directory of `opt` with:
+Also, individual configuration items and the whole configuration can be set using environment variables. For example, the following YAML configuration:
 
-```
-docker compose up
+```yaml
+sources:
+  ldap:
+    url: ldap://localhost:1389
 ```
 
-Or alternatively, without `docker compose`:
+Could be set using the following environment variable:
 
-```
-docker run --rm -it --network host --volume ./opt:/opt/famedly-sync-agent docker-oss.nexus.famedly.de/famedly-sync-agent:latest
+```bash
+FAMEDLY_LDAP_SYNC__SOURCES__LDAP__URL="ldap://localhost:1389"
 ```
 
-### Deactivate only feature
+Note that the environment variable name always starts with the prefix `FAMEDLY_LDAP_SYNC` followed by keys separated by double underscores (`__`).
 
-When this feature is set the ldap-sync will only synchronize the deactivated user. All the changes made on the ldap will be written to the cache as if they where applied. Therefore, only the deactivation changes will be applied to Zitadel but **all the other changes will be lost**
+Some configuration items take a list of values. In this cases the values should be separated by space. **If an empty list is desired the variable shouldn't be created.**
 
-### Configuration
+The main structures is as follows:
 
-The ldap-sync expects a configuration file located at `./config.yaml`. The default path can be changed setting the new path to the environment variable `FAMEDLY_LDAP_SYNC_CONFIG`.
+```
+zitadel
+sources
+  ├── ldap
+  └── ukt
+feature_flags
+```
```

Comment:
The [tree](https://en.wikipedia.org/wiki/Tree_(command)) syntax is a bit misleading here, I'd suggest to use yaml, or actually remove it this paragraph completely. IMO we should either document the whole config structure, or just redirect to .sample.yaml file

Reply:
Okay.